### PR TITLE
Notes for 4.7.24 release

### PR DESCRIPTION
**Pull request by opayne1**
Release notes for 4.7.24 release

- Applies to `enterprise-4.7`
- [Preview Link](https://deploy-preview-35475--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-7-24)

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -2757,3 +2757,28 @@ link:https://access.redhat.com/solutions/6246671[{product-title} 4.7.23 containe
...
+
+[id="ocp-4-7-24-bug-fixes"]
+==== Bug fixes
+
+*  Previously, the installer created out-of-order noProxy values due to allowiuing for spaces in the imput paseed for noProxy. With this update, spaces in the imput were removed to sort the values. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954595[*BZ#1954595*])
+
+*  Previously, `ovn-kubernetes` handled some NetworkPolicies with multiple `ipBlocks` unable to reach all IPs . With this update, generating OVN ACLs from kubernetes NetworkPolicies with multiple `ipBlocks` work correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1967132[*BZ#1967132*])
+
+*  Previously, custome tolerations from `spec.tolerations` were only applied when `spec.nodeSelector` was set. With this update, the operator will use custom tolerations if `spec.tolerations` is set. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1988388[*BZ#1988388*])
+
```
Comment (opayne1) [inline release_notes/ocp-4-7-release-notes.adoc:2780]: PTAL lines 2775-2780 especially. TY!

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -2757,3 +2757,28 @@ link:https://access.redhat.com/solutions/6246671[{product-title} 4.7.23 containe
...
+{product-title} release 4.7.24 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:3032[RHBA-2021:3032] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:3033[RHSA-2021:3033] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6258591[{product-title} 4.7.24 container image list]
+
+[id="ocp-4-7-24-bug-fixes"]
+==== Bug fixes
+
+*  Previously, the installer created out-of-order noProxy values due to allowiuing for spaces in the imput paseed for noProxy. With this update, spaces in the imput were removed to sort the values. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954595[*BZ#1954595*])
```
Comment (EricPonvelle) [inline release_notes/ocp-4-7-release-notes.adoc:2775]: s/allowiuing/allowing

File: release_notes/ocp-4-7-release-notes.adoc [hunk 3]
```diff
@@ -2757,3 +2757,28 @@ link:https://access.redhat.com/solutions/6246671[{product-title} 4.7.23 containe
...
+
+*  Previously, `ovn-kubernetes` handled some NetworkPolicies with multiple `ipBlocks` unable to reach all IPs . With this update, generating OVN ACLs from kubernetes NetworkPolicies with multiple `ipBlocks` work correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1967132[*BZ#1967132*])
```
Comment (EricPonvelle) [inline release_notes/ocp-4-7-release-notes.adoc:2777]: Remove the extra space between IPs and the period.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 4]
```diff
@@ -2757,3 +2757,28 @@ link:https://access.redhat.com/solutions/6246671[{product-title} 4.7.23 containe
...
+
+*  Previously, custome tolerations from `spec.tolerations` were only applied when `spec.nodeSelector` was set. With this update, the operator will use custom tolerations if `spec.tolerations` is set. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1988388[*BZ#1988388*])
```
Comment (EricPonvelle) [inline release_notes/ocp-4-7-release-notes.adoc:2779]: I believe s/custome/custom

**Review by EricPonvelle — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-7-release-notes.adoc (hunk 3 above)
Comment (EricPonvelle) [reply release_notes/ocp-4-7-release-notes.adoc:2777]: s/IPs/IP addresses

File: release_notes/ocp-4-7-release-notes.adoc (hunk 2 above)
Comment (EricPonvelle) [reply release_notes/ocp-4-7-release-notes.adoc:2775]: s/installer/installation program.

s/imput/input (2x)

s/paseed/passed

s/noProxy values/`noProxy` values

File: release_notes/ocp-4-7-release-notes.adoc (hunk 3 above)
Comment (EricPonvelle) [reply release_notes/ocp-4-7-release-notes.adoc:2777]: s/kubernetes NetworkPolicies/Kubernetes NetworkPolicies

File: release_notes/ocp-4-7-release-notes.adoc (hunk 4 above)
Comment (EricPonvelle) [reply release_notes/ocp-4-7-release-notes.adoc:2779]: s/operator/Operator

**Comment by EricPonvelle** [comment]
enterprise-4.7

File: release_notes/ocp-4-7-release-notes.adoc [hunk 5]
```diff
@@ -2757,3 +2757,28 @@ link:https://access.redhat.com/solutions/6246671[{product-title} 4.7.23 containe
...
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6258591[{product-title} 4.7.24 container image list]
+
+[id="ocp-4-7-24-bug-fixes"]
+==== Bug fixes
+
+*  Previously, the installer created out of order `noProxy` values due to allowing for spaces in the input for `noProxy`. With this update, spaces in the input were removed to sort the values. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954595[*BZ#1954595*])
+
+*  Previously, OVN-Kubernetes handled some NetworkPolicies with multiple `ipBlocks` unable to reach all IPs. With this update, generating Open Virtual Network (OVN) ACLs from kubernetes NetworkPolicies with multiple `ipBlocks` work correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1967132[*BZ#1967132*])
```
Comment (EricPonvelle) [inline release_notes/ocp-4-7-release-notes.adoc:2777]: s/IPs/IP addresses

**Review by EricPonvelle — APPROVED** [review]
/lgtm

**Pull request merged**